website/docs/integrations/services/opnsense/index.md

```diff
@@ -0,0 +1,73 @@
+---
+title: OPNsense
+---
+
+## What is OPNsense
+
+From https://opnsense.org/
+
+:::note
+OPNsense is a free and Open-Source FreeBSD-based firewall and routing software. It is licensed under an Open Source Initiative approved license.
+:::
+
+:::note
+This is based on authentik 2021.10.3 and OPNsense 21.7.4-amd64 installed using https://docs.opnsense.org/manual/install.html. Instructions may differ between versions.
+:::
+
+## Preparation
+
+The following placeholders will be used:
+
+- `authentik.company` is the FQDN of authentik.
+- `opnsense-user` is the name of the authentik user we'll create.
+- `DC=ldap,DC=goauthentik,DC=io` is the Base DN of the LDAP Provider (default)
+
+### Step 1
+
+In authentik, under _Identity & Cryptography/Users_, create a user for OPNsense to use as the LDAP Binder.
+
+Change the password of that user by clicking `Impersonate` on the Users page and going to `Change password` after clicking on the settings cog.
```

Comment:
Instead of this way, users should be advised to create a service account which is made specifically for this.

Reply:
Ah yeah, sorted that